src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -72,24 +75,10 @@ private void OnUserSettingsChanged(object sender, EventArgs e)
             }
         }
 
-        private void OnSolutionBindingUpdated(object sender, EventArgs e)
+        private void OnSuppressionsUpdated(object sender, EventArgs e)
         {
-            // NB assumes exception handling is done by the AnalysisRequester
-            if (activeSolutionBoundTracker.CurrentConfiguration.Mode != NewConnectedMode.SonarLintMode.Standalone)
-            {
-                logger.WriteLine(AnalysisStrings.ConfigMonitor_BindingUpdated);
-                analysisRequester.RequestAnalysis();
-            }
-        }
-
-        private void OnSolutionBindingChanged(object sender, ActiveSolutionBindingEventArgs e)
-        {
-            // NB assumes exception handling is done by the AnalysisRequester
-            if (activeSolutionBoundTracker.CurrentConfiguration.Mode != NewConnectedMode.SonarLintMode.Standalone)
-            {
-                logger.WriteLine(AnalysisStrings.ConfigMonitor_SolutionBound);
-                analysisRequester.RequestAnalysis();
-            }
+            logger.WriteLine(AnalysisStrings.ConfigMonitor_BindingUpdated);
```

Comment:
Is this message still appropriate?

Reply:
In a way - it _is_ caused due to updated binding...; but I was more concerned about changing our logging messages: do we do something with them? telemetry?

Reply:
Output messages are entirely separate from telemetry. They are supposed to be meaningful to the end user/us when doing support.

Reporting that the list of suppressions has been synchronized with the connected server might be better since that is all this class knows for sure.